docs/custom-images-contract.rst

```diff
@@ -0,0 +1,113 @@
+.. _custom-images-contract:
+
+======================
+Custom Images contract
+======================
+
+The contract is a set of requirements for crawler custom docker images to comply running with Scrapy Cloud.
+
+The contract is already implemented in Python and is known as :ref:`scrapinghub-entrypoint-scrapy <sh-entrypoint-scrapy>`. If you don’t use this package with your crawler image, you should implement Custom Images contract by yourself, or your image will not work properly in Scrapy Cloud.
```

Comment:
I think that telling that you have to implement the contract if you want to run non-scrapy spiders on Scrapy Cloud is enough.